scripts/extract_validation_messages.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+import yaml, os, shutil
+
+dir = os.path.dirname(__file__)
+
+if os.path.exists('{}/validation_messages'.format(dir)):
+    shutil.rmtree('{}/validation_messages'.format(dir))
+
+os.makedirs('{}/validation_messages'.format(dir))
```

Comment:
All this code which actually *does* something (rather than just defines things) feels like it should belong in the `if __name__ == "__main__"` section. Otherwise simply importing this module from another (for some reason) will cause all sorts of filesystem altering events to happen.